src/utils/api/api.js

```diff
@@ -55,7 +99,12 @@ async function api(
         : ''
     }`,
     options,
-  ).then(res => res.json());
+  ).then(res => {
+    if (options.text) {
+      return res.text();
```

Comment:
Why do we need this? Shouldn't the backend always return json?

Reply:
I added a comment. This is for CSRF token support.